ts/image-occlusion/MaskEditor.svelte

```diff
@@ -32,8 +34,17 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     const startingTool = mode.kind === "add" ? "draw-rectangle" : "cursor";
     $: canvas = null;
 
+    $: {
+        globalThis.maskEditor = canvas ? new MaskEditorAPI(canvas) : null;
+    }
+
     const dispatch = createEventDispatcher();
 
+    function onImageLoaded() {
+        const data = mode.kind === "add" ? mode.imagePath : mode.noteId;
+        bridgeCommand(`ioImageLoaded:${JSON.stringify(data)}`);
```

Comment:
This line may not work on Mobile client because this is specific desktop client.

Reply:
Thanks for taking a look, Mani!

Would this raise an exception in this case, or is `bridgeCommand` just a no-op on mobile? I was assuming the latter, but don't actually know for sure.

Reply:
This will be the error.
```
Uncaught ReferenceError: bridgeCommand is not defined.
```

Reply:
Thanks! I moved the `bridgeCommand` calls to `NoteEditor`, which should fix the mobile compat hopefully.

Reply:
Good catch Mani!